scripts/xtensa-build-zephyr.py

```diff
@@ -138,7 +138,7 @@ class PlatformConfig:
 		f"RG-2017.8{xtensa_tools_version_postfix}",
 		"cavs2x_LX6HiFi3_2017_8",
 		"xcc",
-		aliases = ['adl', 'adl-n', 'ehl', 'rpl'],
+		aliases = ['adl', 'adl-n', 'rpl'],
```

Comment:
Why we pick on EHL, why it is a problem that it is listed, the firmware is the same as tgl/adl/adl-n/rpm. Why is the differentiation?

Reply:
@ujfalusi I want to drop it from sof-bin releases as we have no bandwidth to do signed IPC4 releases for this (and we've been only providing community signed binaries for EHL in the past few release cycles).